group-coordinator/src/test/java/org/apache/kafka/coordinator/group/modern/share/ShareGroupConfigTest.java

```diff
@@ -89,24 +87,6 @@ public void testInvalidConfigs() {
         assertEquals("Invalid value 11 for configuration group.share.delivery.count.limit: Value must be no more than 10",
             assertThrows(ConfigException.class, () -> createConfig(configs)).getMessage());
 
-        configs.clear();
-        // test for when SHARE_GROUP_MAX_GROUPS_CONFIG is of incorrect data type
-        configs.put(ShareGroupConfig.SHARE_GROUP_MAX_GROUPS_CONFIG, 10);
```

Comment:
Should we add unit test for `SHARE_GROUP_MAX_SHARE_SESSIONS_CONFIG`?

Reply:
There is atleast(1) validation itself for the config. And KIP defines no maximum, so do we need any other validation?

Reply:
> Should we increase the min value? Maybe it should be equal to or larger than group.share.max.size?

Yeah, that is something to consider. One should consider to have `group.share.max.share.sessions` >= `group.share.max.size`. @AndrewJSchofield wdyt?

Reply:
@apoorvmittal10 I think that makes a lot of sense.